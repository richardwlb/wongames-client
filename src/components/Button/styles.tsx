import styled from 'styled-components';

export const wrapper = styled.main`
  background-color: #06092b;
`;
