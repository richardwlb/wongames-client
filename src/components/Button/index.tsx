import * as S from './styles';

const Button = () => (
  <S.wrapper>
    <h3>Button</h3>
  </S.wrapper>
);

export default Button;
