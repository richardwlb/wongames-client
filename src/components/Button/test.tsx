import { render, screen } from '@testing-library/react';

import Button from '.';

describe('Button', () => {
  it('Should render the component Button', () => {
    const { container } = render(<Button />);

    expect(
      screen.getByRole('heading', { name: /Button/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
