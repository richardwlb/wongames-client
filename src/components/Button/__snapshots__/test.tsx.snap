// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Should render the component Button 1`] = `
.c0 {
  background-color: #06092b;
}

<main
  class="c0"
>
  <h3>
    Button
  </h3>
</main>
`;
